WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderResourceVars.java

```diff
@@ -29,6 +31,9 @@
         Resources resources = context.getResources();
 
         int displayWidthPx = DisplayUtils.getDisplayPixelWidth(context);
+
+        canShowTiledGallery = DisplayUtils.pxToDp(context, displayWidthPx) > 640;
```

Comment:
In the [description](https://github.com/wordpress-mobile/WordPress-Android/pull/4222#issue-159942596) of this PR it says "less than 620dp" but here it checks for greater than 640dp. Which is correct?